test/addons/at-exit/test.js

```diff
@@ -20,7 +20,7 @@ assert.ifError(p.error);
 assert.deepStrictEqual(
   p.stderr
     .toString()
-    .split('\n')
+    .split(/\n|\r\n?/g)
```

Comment:
NIt:

```suggestion
    .split(/\r?\n/g)
```

Reply:
Or perhaps even better:

```suggestion
    .split(os.EOL)
```